onnxruntime/python/tools/quantization/onnx_quantizer.py

```diff
@@ -694,10 +694,12 @@ def quantize_inputs(self, node, indices, initializer_use_weight_qType=True, redu
 
         return (quantized_input_names, zero_point_names, scale_names, nodes)
 
-    def quantize_weight(self, weight, qType, reduce_range=False):
+    def quantize_weight(self, weight, qType, reduce_range=False, keep_float_weight=False):
         '''
             :param weight: TensorProto initializer
             :param qType: type to quantize to
+            :param keep_float_weight: Whether to quantize the weight. In some cases, we only want to qunatize scale and zero point.
```

Comment:
>we only want to qunatize scale and zero point. [](http://example.com/codeflow?start=85&length=46)

what does quantize scale and zero point mean?

Reply:
https://github.com/microsoft/onnxruntime/blob/5c5fa8246232b8d130a70a466293307440ece682/onnxruntime/python/tools/quantization/onnx_quantizer.py#L716 here in line716, 719 720

Yes, I think I should modify the description to "we only want to calculate scale and zero point"